### PR TITLE
added pulldowns to GPIO pins UART7 RTS and UART7 CTS

### LINKED ISSUE #20762: Pixhawk 6X: MAVLink not working on TELEM1 port
## Describe the bug
On the new Pixhawk 6X, there is no data coming through the TX pin of TELEM1. TELEM3 works, though.

## To Reproduce
1. Hold oscilloscope probe to TX pin of TELEM1 port.
2.  See no data coming through.
3. Hold Probe to TX pin of TELEM3 port.
4. See data.

## Expected behavior
Data coming through TELEM1.

### PR DESCRIPTION
**Pull request by FriedrichTuttas**
### Solved Problem

Fixes #20762

### Solution
As suggested in [this comment](https://github.com/PX4/PX4-Autopilot/issues/20762#issuecomment-1396628897), pulldowns were added to the GPIO pins of UART7 RTS and CTS.

### Alternatives
It was found that [disabling flow control](https://github.com/PX4/PX4-Autopilot/issues/20762#issuecomment-1386351136) or [disabling DMA for UART7](https://github.com/PX4/PX4-Autopilot/issues/20762#issuecomment-1386379337) also enabled MAVLink to work on TELEM1. However, the solution of this PR can leave both flow control and DMA enabled.

### Test coverage
- I tested this code on a Pixhawk 6X. Here is the output of `ver all`

```
nsh> ver all
HW arch: PX4_FMU_V6X
HW type: V6X04
HW version: 0x00000000
HW revision: 0x00000004
FW git-hash: 46a12a09bf11c8cbafc5ad905996645b4fe1a9df
FW version: 1.13.2 0 (17629696)
FW git-branch: iss20762
OS: NuttX
OS version: Release 11.0.0 (184549631)
OS git-hash: 91bece51afbe7da9db12e3695cdbb4f4bba4bc83
Build datetime: Jan 20 2023 11:04:28
Build uri: localhost
Build variant: default
Toolchain: GNU GCC, 9.3.1 20200408 (release)
PX4GUID: 00060000000037343637333051190043003c
MCU: STM32H7[4|5]xxx, rev. V
```

- after flashing the fixed firmware, using an oscilloscope, I can see data on the TX pin of the TELEM1 port with a baud rate of 57600 (as expected for MAVLink messages)
- using [this telemetry module](https://www.mybotshop.de/Drotek-Radio-Telemetry-433Mhz-3DR) the Pixhawk 6X now connects to QGC

### Context
[Issue 20763: Pixhawk 6X: MAVLink not working on TELEM1 port](https://github.com/PX4/PX4-Autopilot/issues/20762)

### CODE REVIEW

File: boards/px4/fmu-v6x/nuttx-config/include/board.h [hunk 1]
```diff
@@ -387,8 +387,8 @@
 
 #define GPIO_UART7_RX    GPIO_UART7_RX_4    /* PF6  */
 #define GPIO_UART7_TX    GPIO_UART7_TX_3    /* PE8  */
-#define GPIO_UART7_RTS   GPIO_UART7_RTS_2   /* PF8  */
-#define GPIO_UART7_CTS   GPIO_UART7_CTS_1   /* PE10 */
+#define GPIO_UART7_RTS   (GPIO_UART7_RTS_2|GPIO_PULLDOWN)   /* PF8  */
```
Comment (davids5) [inline boards/px4/fmu-v6x/nuttx-config/include/board.h:390]:
```suggestion
#define GPIO_UART7_RTS   (GPIO_UART7_RTS_2 | GPIO_PULLDOWN)   /* PF8  */
```

File: boards/px4/fmu-v6x/nuttx-config/include/board.h [hunk 2]
```diff
@@ -387,8 +387,8 @@
...
+#define GPIO_UART7_CTS   (GPIO_UART7_CTS_1||GPIO_PULLDOWN)   /* PE10 */
```
Comment (davids5) [inline boards/px4/fmu-v6x/nuttx-config/include/board.h:391]:
```suggestion
#define GPIO_UART7_CTS   (GPIO_UART7_CTS_1 | GPIO_PULLDOWN)   /* PE10 */
```

**Review by davids5 — CHANGES_REQUESTED** [review]
@FriedrichTuttas - Thank you. Some style and an error fixed.

File: boards/px4/fmu-v6x/nuttx-config/include/board.h (hunk 2 above)
Comment (FriedrichTuttas) [reply boards/px4/fmu-v6x/nuttx-config/include/board.h:391]: Oh, I completely missed that I typed two "|". Thanks for the correction.

**Review by davids5 — APPROVED** [review]
@FriedrichTuttas Looks good!

**Comment by dagar** [comment]
Is there a reason it's not done for USART2 (telem3) as well? 

https://github.com/PX4/PX4-Autopilot/blob/42f44f5d9ed663a863aec9fc9d093a9e5c18a985/boards/px4/fmu-v6x/nuttx-config/include/board.h#L369-L372

**Comment by davids5** [comment]
@dagar No. It should probably be done on all the ports and platforms.

**Comment by julianoes** [comment]
@davids5, right, so we should follow that up in a future PR for all other boards and ports.

**Comment by julianoes** [comment]
And this fix here should probably be backported to v1.13/

**Pull request merged**